### ISSUE TITLE
[ENHANCEMENT] Disable avahi-daemon by default

### ISSUE THREAD

**Issue by bk201** [harvester/harvester#3573]
**Is your enhancement related to a problem? Please describe.**

Disable avahi-daemon by default, it not needed.

**Describe the solution you'd like**

Disable avahi-daemon by default.

**Describe alternatives you've considered**

**Additional context**

**Comment by irishgordo** [comment]
Possible similar issue - seems blocked for testing currently:
https://github.com/harvester/harvester/issues/3584

**Comment by TachunLin** [comment]
Verified **fixed** on `master-e341a72d-head` (03/06). Close this issue.

### Result
The avahi-daemon service have been disabled on each node of the three nodes Harvester cluster 
```
n1-230306:~ # sudo systemctl status avahi-daemon.service
○ avahi-daemon.service - Avahi mDNS/DNS-SD Stack
     Loaded: loaded (/usr/lib/systemd/system/avahi-daemon.service; disabled; vendor preset: enabled)
     Active: inactive (dead)
TriggeredBy: ○ avahi-daemon.socket

n2-230306:~ # sudo systemctl status avahi-daemon.service
○ avahi-daemon.service - Avahi mDNS/DNS-SD Stack
     Loaded: loaded (/usr/lib/systemd/system/avahi-daemon.service; disabled; vendor preset: enabled)
     Active: inactive (dead)
TriggeredBy: ○ avahi-daemon.socket

n3-230306:~ # sudo systemctl status avahi-daemon.service
○ avahi-daemon.service - Avahi mDNS/DNS-SD Stack
     Loaded: loaded (/usr/lib/systemd/system/avahi-daemon.service; disabled; vendor preset: enabled)
     Active: inactive (dead)
TriggeredBy: ○ avahi-daemon.socket

```

### Test Information
* Test Environment: 3 nodes harvester on local kvm machine
* Harvester version: `master-e341a72d-head` (03/06)

### Verify Steps
1. ssh to each node of Harvester cluster 
2. Execute command `systemctl status avahi-daemon.service`
